omniport/core/open_auth/serializers/application.py

```diff
@@ -215,3 +215,38 @@ def update(self, instance, validated_data):
         application = super().update(instance, validated_data)
 
         return application
+
+
+class ApplicationHiddenDetailSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Application objects that exposes all information
+    """
+
+    team_members = PersonRelatedField(
+        queryset=Person.objects.all(),
+        many=True,
+        required=False,
+    )
+
+    class Meta:
+        """
+        Meta class for ApplicationDetailSerializer
+        """
+
+        model = Application
+        read_only_fields = [
+            'client_secret', 
+            'client_id',           
```

Comment:
Why are we sending `client_id` as well? We don't need to hide that. In my opinion, the view should only serve `client_secret`.